pinot-core/src/main/java/com/linkedin/pinot/core/segment/index/loader/invertedindex/InvertedIndexHandler.java

```diff
@@ -68,7 +69,11 @@ public void createInvertedIndices()
     Set<String> invertedIndexColumns = getInvertedIndexColumns();
 
     for (String column : invertedIndexColumns) {
-      createInvertedIndexForColumn(segmentMetadata.getColumnMetadataFor(column));
+      try {
+        createInvertedIndexForColumn(segmentMetadata.getColumnMetadataFor(column));
+      } catch (Exception e) {
+        LOGGER.warn("Inverted index not created for column {}", column);
```

Comment:
Do we want to swallow this exception?

Reply:
Yes, so that we can try to build inverted index for the next field. Not building inverted index is not an error, so no need to abort here.